cristin-person/src/test/java/no/unit/nva/cristin/person/handler/PersonQueryHandlerTest.java

```diff
@@ -58,6 +82,50 @@ void shouldReturnResponseWhenCallingEndpointWithNameParameter() throws IOExcepti
         assertEquals(expected, actual);
     }
 
+    @Test
+    void shouldThrowCorrectErrorResponseToClientWhenBackendFetchFails() throws IOException {
```

Comment:
I am not sure this is as broad as it claims to be. It only tests the case where a RuntimeException produces a Runtime error. If there are more cases I suggest writing a test for each case. If there aren't more cases then rename the test

Reply:
Updated name and added extra test.